clients/idms-dialect-support/README.md

```diff
@@ -22,11 +22,12 @@ IDMS Dialect Support enables the following LSP features:
 - Autocomplete for IDMS DML keywords
 - Syntax and semantic check for IDMS DML keywords and variables
 - Syntax highlighting and coloring for IDMS DML keywords and variables
+- IDMS code snippets
 
 ## Copybook Support
 
-IDMS Dialect Support enables copybook support features of the COBOL Language Support extension for copybooks called by the `COPY IDMS` statement. 
+IDMS Dialect Support enables copybook support features of the COBOL Language Support extension for copybooks named in `COPY IDMS` statements. 
 
 To enable copybook support, follow the instructions in the [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) documentation. 
 
-Specify your IDMS copybooks in the COBOL Language Support extension settings under **Cpy-manager: Paths-local: Idms**.
+Specify your local IDMS copybook paths in the COBOL Language Support extension settings under **Cpy-manager: Paths-local: Idms**. Automatic retrieval of IDMS copybooks from the mainframe is also supported.
```

Comment:
This setting will be from now on part of the addon settings. All three options will be available: local, DNS and USS. But not in COBOL LS setting section

Reply:
<img width="840" alt="Screenshot 2023-03-01 at 17 58 55" src="https://user-images.githubusercontent.com/38490125/222210151-b8d20bc2-cc7d-4fea-ada6-62af5a2c84fc.png">

Reply:
Thanks Lukas this is fixed now